lib/pbench/test/unit/server/query_apis/test_datasets_detail.py

```diff
@@ -205,7 +207,7 @@ def test_empty_query(
         fixture.
         """
         expected_status = HTTPStatus.BAD_REQUEST
-        if build_auth_header["header_param"] != "valid":
+        if build_auth_header["header_param"] not in ["valid", "valid_admin"]:
```

Comment:
This conditional looks ripe for abstracting into a helper function -- the calling code wants to know if the header is valid, and it doesn't necessarily need to know the details of what makes it so.